test/image-blueprints/layer3-periodic/group1/rhel92-prel-with-optionals.toml

```diff
@@ -0,0 +1,66 @@
+{{- if env.Getenv "PREVIOUS_RELEASE_VERSION" "" -}}
+{{- /*
+
+  We wrap this template in a test so that the body of the output is
+  empty when there is no "current" version release. The output file
+  must end up completely empty, so we need to remove whitespace from
+  around the first and last template instructions.
+
+*/ -}}
+
+name = "rhel-9.2-microshift-prel-optionals"
+description = "A RHEL 9.2 image with already built and released RPMs like EC, RC, or Z-stream release"
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-92"
+
+[[packages]]
+name = "microshift"
+version = "{{ .Env.PREVIOUS_RELEASE_VERSION }}"
+
+[[packages]]
+name = "microshift-greenboot"
+version = "{{ .Env.PREVIOUS_RELEASE_VERSION }}"
+
+[[packages]]
+name = "microshift-networking"
+version = "{{ .Env.PREVIOUS_RELEASE_VERSION }}"
+
+[[packages]]
+name = "microshift-selinux"
+version = "{{ .Env.PREVIOUS_RELEASE_VERSION }}"
+
+[[packages]]
+name = "microshift-olm"
+version = "{{ .Env.PREVIOUS_RELEASE_VERSION }}"
+
+[[packages]]
+name = "microshift-multus"
```

Comment:
We don't have multus in 4.15. We're after branching, but the `Makefile.version.*.var` still says 4.16.
I guess we need to postpone "prel->src" optional upgrades